### PR TITLE
Add new role: OBO Code of Coduct Committee Member

### LINKED ISSUE #2361: Formally registering the COC committee
Summary of OFOC mailing list:

1. We have a code of conduct (COC): https://obofoundry.org/docs/COC.html
1. We don't have the COC team documented anywhere. These should be documented in the usual way in [roles](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/roles.yml), but we also need to list members, I think, as a separate group in [operations](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml)
1. The COC team should be diverse (min 5 members?), and each team member should be individually contactable by community members (@mellybelly says: `The main considerations are that it should contain several diverse people so that community members can choose to consult with someone they are most comfortable with, and that the committee must have a confidential process for reconciling issues and the authority to do so.`)
1. There should be at least a basic SOP that is concerned with documenting confidentiality (not revealing the source of the complaints), and modus operandi when a complaint is received.

cc @bpeters42, @hoganwr who helped jump-starting the discussion again

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #2361, #1605

@lschriml, @addiehl as you are so far the only volunteers, I tag you specifically. Can you take a look at the first draft of the role description here, and let me know what you think.

- [x] Update the link from COC to the committee member pages
- [x] Add emails of COC committee members to the site

### CODE REVIEW

File: _data/roles.yml [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+    They are, in particular, available for anyone in the community to be contacted individually and as a group.
+  commitment: 0-6 hours per month
+  requirements:
+    - being able to deal with sensitive interpersonal issues discretely
+    - being willing to mediate between conflicting viewpoints
+    - being highly available when issues arise
+    - fully embracing the OBO Code of Conduct
+  responsibilities:
+    - Being available for receiving COC-related issues via email from anyone in the community, and dealing with them discretely (not sharing any part of the communication without permission)
+    - Developing strategies for dealing with COC-related issues which are often socio-technologically complex
```
[the remark this answers is not in the document]
Comment (matentzn) [reply _data/roles.yml:14]: Absolutely, what about adding a phrase here like: 

dealing... according to established COC protocols?

File: _data/roles.yml [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+    - Being available to receive COC-related issues via email from anyone in the community, and dealing with them discreetly (not sharing any part of the communication without permission)
+    - Developing strategies for dealing with COC-related issues which are often socio-technologically complex
+    - Ensuring that the OBO Code of Conduct is upheld around matters directly related to the OBO Foundry
+    - (Other responsibilities are yet to be defined for this role)
+  people:
+    - github: addiehl
+      name: Alexander Diehl
+      orcid: 0000-0001-9990-8331
+      status: support
+      start: 2023-05-16
```
Comment (cthoyt) [inline _data/roles.yml:22]:
```suggestion
      start: "2023-05-16"
```

File: _data/roles.yml [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+    - github: lschriml
+      name: Lynn Schriml
+      orcid: 0000-0001-8910-9851
+      status: support
+      start: 2023-05-16
```
Comment (cthoyt) [inline _data/roles.yml:27]:
```suggestion
      start: "2023-05-16"
```

File: _data/roles.yml [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+    - github: matentzn
+      name: Nico Matentzoglu
+      orcid: 0000-0002-7356-1779
+      status: support
+      start: 2023-06-15
```
Comment (cthoyt) [inline _data/roles.yml:32]:
```suggestion
      start: "2023-05-16"
```

File: _data/roles.yml (hunk 2 above)
Comment (cthoyt) [reply _data/roles.yml:22]: I can change the code handling of these in a different PR, but just accept these changes to make the PR pass the tests

@matentzn

Comment (cthoyt) [reply _data/roles.yml:22]: also please double check the start dates, they are not consistent. some say 2023-05-16 and some say 2023-06-16.

Comment (matentzn) [reply _data/roles.yml]: I used the date whatever person declared their wish to participate, but I don't think the exact date matters that much.. Its fine like you suggest.

**Comment by nlharris** [comment]
What's the status of this?

**Review by lschriml — APPROVED** [review]
Looks good to me.
Lynn

**Review by addiehl — APPROVED** [review]

**Pull request merged**